cni/network/multitenancy.go

```diff
@@ -199,60 +311,60 @@ func checkIfSubnetOverlaps(enableInfraVnet bool, nwCfg *cni.NetworkConfig, cnsNe
 	return false
 }
 
+var (
+	errSnatIP        = errors.New("Snat IP not populated")
+	errInfraVnet     = errors.New("infravnet not populated")
+	errSubnetOverlap = errors.New("subnet overlap error")
+)
+
 // GetMultiTenancyCNIResult retrieves network goal state of a container from CNS
-func GetMultiTenancyCNIResult(
+func (plugin *NetPlugin) GetMultiTenancyCNIResult(
 	ctx context.Context,
 	enableInfraVnet bool,
 	nwCfg *cni.NetworkConfig,
-	plugin *netPlugin,
 	k8sPodName string,
 	k8sNamespace string,
-	ifName string) (*cniTypesCurr.Result, *cns.GetNetworkContainerResponse, net.IPNet, *cniTypesCurr.Result, error) {
+	ifName string) (res *cniTypesCurr.Result, resp *cns.GetNetworkContainerResponse, prefix net.IPNet, infraRes *cniTypesCurr.Result, err error) {
 
-	if nwCfg.MultiTenancy {
-		result, cnsNetworkConfig, subnetPrefix, err := getContainerNetworkConfiguration(ctx, nwCfg, k8sPodName, k8sNamespace, ifName)
-		if err != nil {
-			log.Printf("GetContainerNetworkConfiguration failed for podname %v namespace %v with error %v", k8sPodName, k8sNamespace, err)
-			return nil, nil, net.IPNet{}, nil, err
-		}
+	result, cnsNetworkConfig, subnetPrefix, err := plugin.multitenancyClient.GetContainerNetworkConfiguration(ctx, nwCfg, k8sPodName, k8sNamespace, ifName)
+	if err != nil {
+		log.Printf("GetContainerNetworkConfiguration failed for podname %v namespace %v with error %v", k8sPodName, k8sNamespace, err)
+		return nil, nil, net.IPNet{}, nil, fmt.Errorf("%w", err)
```

Comment:
formatting directive here should provide context, this defeats the purpose of the wrap